Privacy changes

masterlittle opened the pull request:
## What type of PR is this? (check all applicable)
<!-- Please leave only what's applicable -->

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

## Description

## Related Tickets & Documents

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Commit:
Add models for pii data

Commit:
Add policy tag checks and additional tables in redash db

Commit:
Fix user permissions

avinashdunzo commented on redash/models/__init__.py:
```diff
@@ -82,6 +81,69 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+class PolicyTag(db.Model):
```
How is migration being run? Is it running automatically?

masterlittle replied:
Will have to run these manually right now.
1. Not very familiar with alembic.
2. These are all new tables so nothing will break.

For any new setup in a db, these will be created automatically.

avinashdunzo replied:
Then checkin a SQL file. will all the create table queries.

masterlittle replied:
Yes. Will do

Commit:
Fix query_id change

Commit:
Add DDL statement file

Commit:
Remove foreign key constraints from privacy tables

Commit:
Update docker compose file

Commit:
Add datatype to bigquery column name
Enable auto limit for bigquery
Show slots in bigquery queries in UI
Fix docker-compose.yml by adding postgres container name

Commit:
Add component to show all policy tags accessed

Commit:
Add functionality to use bigquery parser for pii

Commit:
Add formatting changes
